django/cantusdb_project/main_app/templates/chant_edit.html

```diff
@@ -236,13 +236,17 @@ <h3>Full text &amp; Volpiano edit form</h3>
                     <dd><small>4) Edit the fields <b>according to the manuscript, following the fulltext guidelines created by Cantus</b></small></dd>
                     <dd><small>5) Click <b>"SAVE"</b></small></dd>
                 </dl> 
-                <a href="{% url "source-create" %}" style="display: inline-block; margin-top:5px;">
-                    <small><b>&plus; Add new source</b></small>
-                </a>
-                <br>
                 <a href="{% url 'source-detail' source.id %}" title="{{ source.title }}" style="display: inline-block; margin-top:5px;">
                     {{ source.siglum }}
                 </a>
+                <br>
+                <a href="{% url "chant-create" source.pk %}" style="display: inline-block; margin-top:5px;">
```

Comment:
why are we using both `<br>` and `margin-top`? It might be cleaner to put these links in a `<div>` (each link might need its own div - not sure) and apply `margin-top` to each. In which case, we may not need `inline-block` either...